From: Sebastien Bacher <seb128@ubuntu.com>
Date: Sun, 18 Feb 2018 13:07:32 -0500
Subject: Have the HighContrast theme inherit icons from Humanity

Otherwise lots of icons would be missing, especially since Ubuntu
doesn't install either gnome-icon-theme or adwaita-icon-theme-full

Forwarded: not-needed
Bug: https://launchpad.net/bugs/1018535
---
 themes/HighContrast/icons/index.theme.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/themes/HighContrast/icons/index.theme.in b/themes/HighContrast/icons/index.theme.in
index e2a14b6..28ac1e3 100644
--- a/themes/HighContrast/icons/index.theme.in
+++ b/themes/HighContrast/icons/index.theme.in
@@ -1,7 +1,7 @@
 [Icon Theme]
 Name=HighContrast
 Comment=High contrast scalable icons with black borders
-Inherits=Adwaita
+Inherits=Adwaita,gnome,Humanity
 Example=folder
 
 Directories=@THEME_DIRS@
